docs/projects/ansible/collection/centurion/playbooks/index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Playbooks
+description: No Fuss Computings Companion Ansible Collection Centurion Playbooks.
```

Comment:
the what now??

Reply:
Still need to adjust this

Reply:
Well thats what they are NFC's companion ansible collection centurion playbooks

Reply:
```suggestion
description: Playbooks as part of No Fuss Computings Companion Ansible Collection, Centurion .
```

Reply:
fixed in commit [054c201](https://github.com/nofusscomputing/ansible_collection_centurion/pull/17/commits/054c201d1abd6accf7ee94843a472849075bd5e1)